### PR TITLE
Dev branch

### PR DESCRIPTION
**Pull request by elimc**
This is just a test.

### CODE REVIEW

**Comment by elimc** [comment]
elimc is a very serious coder, with a mysterious past . . .

File: test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hello world. This is the dev.
\ No newline at end of file
```
Comment (elimc) [inline test.txt:1]: Just a small note

**Comment by elimc** [comment]
Github is very confusing, for such a simple interface.

File: test.txt (hunk 1 above)
Comment (yatharth) [inline test.txt:1]: Why does it show this (`\ No newline at the end of the file`)?

**Pull request closed without merging**